src/math/numbers.c

```diff
@@ -129,3 +129,22 @@ int norm_int32(int32_t val)
 	}
 	return s;
 }
+
+uint32_t crc32(const void *data, uint32_t size)
```

Comment:
Can we change size to bytes since it looks like there is no dependency that the size being 32bit aligned ?